packages/main/cypress/specs/TableSelection.cy.tsx

```diff
@@ -0,0 +1,291 @@
+import Label from "../../src/Label.js";
+import Table from "../../src/Table.js";
+import TableHeaderRow from "../../src/TableHeaderRow.js";
+import TableHeaderCell from "../../src/TableHeaderCell.js";
+import TableRow from "../../src/TableRow.js";
+import TableCell from "../../src/TableCell.js";
+import TableSelectionMode from "../../src/types/TableSelectionMode.js";
+import TableSelection from "../../src/TableSelection.js";
+
+function changeMode(mode: TableSelectionMode) {
```

Comment:
this method always used after the mountTestpage so it can be the part of the mountTestpage